From 263bd26b1216c933db3d216197a78678d0f8610e Mon Sep 17 00:00:00 2001
From: Libin Yang <libin.yang@intel.com>
Date: Thu, 19 Nov 2020 20:01:30 +0800
Subject: [PATCH] rt715: init: setup ADC07 to a proper volume

This patch initiates 'rt715 ADC 07 Capture Volume' to a proper volume.

Signed-off-by: Libin Yang <libin.yang@intel.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/codecs/rt715/init.conf | 1 +
 1 file changed, 1 insertion(+)

Index: alsa-ucm-conf-1.2.4/ucm2/codecs/rt715/init.conf
===================================================================
--- alsa-ucm-conf-1.2.4.orig/ucm2/codecs/rt715/init.conf
+++ alsa-ucm-conf-1.2.4/ucm2/codecs/rt715/init.conf
@@ -7,4 +7,5 @@ BootSequence [
 	cset "name='rt715 ADC 25 Mux' 4"
 	cset "name='rt715 ADC 27 Capture Switch' 1"
 	cset "name='rt715 ADC 07 Capture Switch' 1"
+	cset "name='rt715 ADC 07 Capture Volume' 58"
 ]
